docs/README.md

```diff
@@ -23,35 +23,107 @@ WIP
 
```

Comment:
This is actually for the line above:

2. Now you can use the two blocks exported by `vtex.modal-layout`. Notice that you need to configure your own `modal-layout` and pass it as a child of `modal-trigger`.

Reply:
Also, for line 14, add the `"dependencies"` part to it:

```json
{
  "dependencies": {
      "vtex.modal-layout": "0.x"
  }
}
```